ush/parsing_namelists_FV3.sh

```diff
@@ -0,0 +1,603 @@
+  lheatstrg    = ${lheatstrg-".false."}
+  random_clds  = ${random_clds:-".true."} 
+  trans_trac   = ${trans_trac:-".true."}
+  cnvcld       = ${cnvcld:-".true."}
+  imfshalcnv   = ${imfshalcnv:-"2"}
+  imfdeepcnv   = ${imfdeepcnv:-"2"}
+  cdmbgwd      = ${cdmbgwd:-"3.5,0.25"}   
+  prslrd0      = ${prslrd0:-"0."}
+  ivegsrc      = ${ivegsrc:-"1"}
+  isot         = ${isot:-"1"}
+  lsoil        = ${lsoil:-"4"}
+  lsm          = ${lsm:-"2"}
+  iopt_dveg    = ${iopt_dveg:-"1"}
+  iopt_crs     = ${iopt_crs:-"1"}
+  iopt_btr     = ${iopt_btr:-"1"}
+  iopt_run     = ${iopt_run:-"1"}
+  iopt_sfc     = ${iopt_sfc:-"1"}
+  iopt_frz     = ${iopt_frz:-"1"}
+  iopt_inf     = ${iopt_inf:-"1"}
+  iopt_rad     = ${iopt_rad:-"1"}
+  iopt_alb     = ${iopt_alb:-"2"}
+  iopt_snf     = ${iopt_snf:-"4"}
+  iopt_tbot    = ${iopt_tbot:-"2"}
+  iopt_stc     = ${iopt_stc:-"1"}
+  debug        = ${gfs_phys_debug:-".false."}
+  nstf_name    = $nstf_name
+  nst_anl      = $nst_anl           
+  psautco      = ${psautco:-"0.0008,0.0005"}
+  prautco      = ${prautco:-"0.00015,0.00015"}
+  lgfdlmprad   = ${lgfdlmprad:-".false."}
+  effr_in      = ${effr_in:-".false."}
+  cplwav       = ${cplwav:-".false."}        
+  ldiag_ugwp   = ${ldiag_ugwp:-".false."}
+  do_ugwp      = ${do_ugwp:-".true."}
+EOF
+
+if [ $cpl = .true. ]; then
+  cat >> input.nml << EOF
+  do_tofd      = ${do_tofd:-".false."}
+  fscav_aero = ${fscav_aero:-'*:0.0'}
+EOF
+else
+cat >> input.nml <<EOF
+  do_tofd      = ${do_tofd:-".true."}
+EOF
+fi
+
+cat >> input.nml <<EOF
+  do_sppt      = ${DO_SPPT:-".false."}
+  do_shum      = ${DO_SHUM:-".false."}
+  do_skeb      = ${DO_SKEB:-".false."}
+EOF
+
+if [ $cpl = .true. ]; then
+  cat >> input.nml << EOF
+  frac_grid    = ${FRAC_GRID:-".true."}
+  cplchm = ${cplchem:-".false."}
+  cplflx       = $cplflx
+  cplwav2atm   = ${cplwav2atm}       
+EOF
+fi
+
+# Add namelist for IAU
+if [ $DOIAU = "YES" ]; then
+  cat >> input.nml << EOF
+  iaufhrs      = ${IAUFHRS}
+  iau_delthrs  = ${IAU_DELTHRS}
+  iau_inc_files= ${IAU_INC_FILES}
+  iau_drymassfixer = .false.
+EOF
+fi
+
+case ${gwd_opt:-"2"} in
+  1)
+  cat >> input.nml <<EOF
+  gwd_opt      = 1
+  do_ugwp      = .false.
+  do_ugwp_v0   = .false.
+  do_ugwp_v1   = .false.
+  do_tofd      = .true.
+  $gfs_physics_nml
+/
+&cires_ugwp_nml
+  knob_ugwp_version = ${knob_ugwp_version:-0}
+  knob_ugwp_solver  = ${knob_ugwp_solver:-2}
+  knob_ugwp_source  = ${knob_ugwp_source:-1,1,0,0}
+  knob_ugwp_wvspec  = ${knob_ugwp_wvspec:-1,25,25,25}
+  knob_ugwp_azdir   = ${knob_ugwp_azdir:-2,4,4,4}
+  knob_ugwp_stoch   = ${knob_ugwp_stoch:-0,0,0,0}
+  knob_ugwp_effac   = ${knob_ugwp_effac:-1,1,1,1}
+  knob_ugwp_doaxyz  = ${knob_ugwp_doaxyz:-1}
+  knob_ugwp_doheat  = ${knob_ugwp_doheat:-1}
+  knob_ugwp_dokdis  = ${knob_ugwp_dokdis:-1}
+  knob_ugwp_ndx4lh  = ${knob_ugwp_ndx4lh:-1}
+  launch_level      = ${launch_level:-54}
+  $cires_ugwp_nml
+/
+
+EOF
+  ;;
+  2)
+  cat >> input.nml << EOF
+  gwd_opt      = 2
+  do_ugwp      = .true.
```

Comment:
This line showed up as a diff from f/c-c but maybe it's okay as is.  @WalterKolczynski-NOAA can you weigh in on that?

Reply:
Should be false. It doesn't make sense to me, but that's what is in the RT.

Reply:
This still needs to be changed back to .false.